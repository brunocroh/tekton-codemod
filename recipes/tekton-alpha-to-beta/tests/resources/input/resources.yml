spec:
  inputs:
    resources:
      - name: skaffold
        type: git
  outputs:
    resources:
      - name: baked-image
        type: image
