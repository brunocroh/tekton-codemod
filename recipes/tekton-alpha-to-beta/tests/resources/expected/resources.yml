spec:
  resources:
    inputs:
      - name: skaffold
        type: git
    outputs:
      - name: baked-image
        type: image

  